docs/Testing.md

```diff
@@ -19,6 +19,7 @@ cmake --build . --config Release --target check-onnx-backend[-jni]
 ``` 
 Packages, such as third_party/onnx, needs to be installed to run the backend test. JNI test requires the jsoniter jar which is downloaded from its maven repository by default if no installed version is found on the system. If the user turns on the cmake option `ONNX_MLIR_BUILD_JSONITER` when building ONNX-MLIR, the jsoniter jar will be built locally from the source cloned from its github repository. Note that building jsoniter jar locally requires the maven build tool to be installed.
```

Comment:
Do we have a link on how third_party/onnx is installed? I did not find any help on this. Some unrelated site to us mention on mac to use `pip install -e third_party/onnx` which is what I do too.

I would add "[[In addition,  ]]JNI test requires the jointer jar" to be a bit more explicit that this is in addition to the one before.

How is the `test/backend/all_test_names.txt` file generated. Custom target or each time?

Reply:
I do not use -e. Here is the explanation: https://stackoverflow.com/questions/41535915/python-pip-install-from-local-dir